### PR TITLE
BAU: Add a script to automatically build .env file for a given target environment

### PR DESCRIPTION
**Pull request by whi-tw**
## What

Create a script which enables the auto-generation of a .env file for a provided target environment.

This will stop us from having to create, store and distribute multiple versions for each environment.

## How to review

- Code Review

1. remove your existing .env file (`mv .env .env.original`)
2. run the script `scripts/create-env-file build`. Observe that a new .env file has been created
3. run the frontend application `./startup.sh -lc`.
4. Access via the local stub (https://localhost:2000) Observe that there are no errors and you are redirected to the frontend
5. modify one of the variables above the 'DO NOT MODIFY AFTER THIS POINT' marker
6. run the script `scripts/create-env-file build`. Observe that your changed variable is still set
7. Add a new variable, rerun the script, observe that your new variable is still present under a 'unknown variables' header

8. Repeat the above steps for `sandpit, authdev#`

### CODE REVIEW

File: build-env.sh.bak [hunk 1]
```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
```
Comment (dbes-gds) [inline build-env.sh.bak:1]: did you mean to add this?

**Comment by dbes-gds** [comment]
something in the readme to tell people how to use this?

File: build-env.sh.bak (hunk 1 above)
Comment (whi-tw) [reply build-env.sh.bak:1]: i did not lol

**Comment by BeckaL** [comment]
All the steps tested as per the instructions here, all looking good. Will code review now

**Review by dbes-gds — APPROVED** [review]

**Review by BeckaL — APPROVED** [review]
This will save an unbelievable amount of faff and time, thank you! Looks good to me

**Comment by whi-tw** [comment]
ticket created: AUT-2759

**Pull request merged**